### PR TITLE
Fe 14 create user form

### PR DESCRIPTION
**Pull request by Connor-Quinny**
-Tailwind css set up
-React native navigation set up
-Firebase connected 
-Sign up form completed
-Form data sent to firebase

### CODE REVIEW

**Comment by BrokenRecord85** [comment]
All good

**Review by BrokenRecord85 — APPROVED** [review]

File: src/components/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded-full-5"
+          />
+        </View>
+      </View>
+    </SafeAreaView>
+  );
+};
+
+{
+  /* <TextInput
```
Comment (MartinSWDev) [inline src/components/Home.jsx]: Do we need these comments?

**Review by MartinSWDev — APPROVED** [review]
Just some comments that might need removing otherwise fine

File: src/components/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+style={styles.input}
+onChangeText={onChangeNumber}
+value={number}
+placeholder="useless placeholder"
+keyboardType="numeric"
+/> */
+}
+
+// const styles = StyleSheet.create({
+// input: {
```
Comment (h-chagas) [inline src/components/Home.jsx]: Just get rid of these comments

**Review by h-chagas — COMMENTED** [review]
Great. Just make sure to delete comments on Home components

File: src/components/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  // Initialize Firebase
+  const app = initializeApp(firebaseConfig);
+  const db = initializeFirestore(app, {
+    experimentalForceLongPolling: true,
+    useFetchStreams: false,
+  });
+
+  const addUser = () => {
+    console.log(username);
+    console.log(email);
```
Comment (Zakkiassi) [inline src/components/Home.jsx]: remove console.logs

**Review by h-chagas — APPROVED** [review]

File: src/components/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+          />
+        </View>
+      </View>
+    </SafeAreaView>
+  );
+};
+
+{
+  /* <TextInput
+style={styles.input}
```
Comment (Zakkiassi) [inline src/components/Home.jsx]: remove commented code

**Review by Zakkiassi — APPROVED** [review]

**Pull request merged**